README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Setup
 
-Ideally this website should be ran through a sub-folder of [Strata](https://github.com/CocoaPods/Strata).
+Ideally this website should be run through a sub-folder of [Strata](https://github.com/CocoaPods/Strata).
```

Comment:
hahaha, my girlfriend gets really annoyed with this one, I do it all the time, it's how we say it in my accent :dancers: